src/Illuminate/Support/Arr.php

```diff
@@ -601,6 +601,10 @@ public static function where($array, callable $callback)
      */
```

Comment:
Change comment: "If the given value is not an array, wrap it in one." is no longer true in this case.

Reply:
Done ☺️